htdocs/societe/class/societe.class.php

```diff
@@ -1924,6 +1929,12 @@ function add_commercial(User $user, $commid)
 			{
 				dol_syslog(get_class($this)."::add_commercial Erreur");
 			}
+			else {
+				$this->commercial_modified = $commid;
+				
+				$result=$this->call_trigger('COMPANY_ADD_COMMERCIAL',$user);
```

Comment:
Can you name your trigger 'COMPANY_LINK' to match naming rules used by other objects